client/src/components/UMDAnnotation.vue

```diff
@@ -98,7 +106,7 @@ export default defineComponent({
       emotionsList?: string[];
       multiSpeaker?: 'FALSE' | 'TRUE' | 'noann';
       normsSelected?: string[];
-      normsObject?: Record<string, 'adhered' |'violate' | 'noann' | 'EMPTY_NA'>;
+      normsObject?: Record<string, 'adhered' |'violated' | 'noann' | 'adhered_violated'>;
```

Comment:
Let's factor out this type definition out since it appears twice in the code (line 100 above).